assets/js/components/notifications/FirstPartyModeSetupBanner.stories.js

```diff
@@ -48,11 +49,59 @@ function Template() {
 export const Default = Template.bind();
 Default.storyName = 'FirstPartyModeSetupBanner';
 Default.scenario = {};
+Default.args = {
+	setupRegistry: () => {
+		fetchMock.post(
+			new RegExp( '^/google-site-kit/v1/core/site/data/fpm-settings' ),
+			{
+				body: JSON.stringify( {
+					isEnabled: true,
+					isFPMHealthy: true,
+					isScriptAccessEnabled: true,
+				} ),
+				status: 200,
+			}
+		);
+	},
+};
+
+export const ErrorOnCTAClick = Template.bind();
+ErrorOnCTAClick.storyName = 'ErrorOnCTAClick';
+ErrorOnCTAClick.scenario = {};
+ErrorOnCTAClick.args = {
+	setupRegistry: ( registry ) => {
+		fetchMock.post(
+			new RegExp( '^/google-site-kit/v1/core/site/data/fpm-settings' ),
+			{
+				body: JSON.stringify( {
+					code: 'test_error',
+					message: 'Test Error',
+					data: {
+						reason: 'test_reason',
+					},
+				} ),
+				status: 500,
+			}
+		);
```

Comment:
This `fetchMock` is unnecessary since we are mocking the error using `receiveError` below.
```suggestion
```